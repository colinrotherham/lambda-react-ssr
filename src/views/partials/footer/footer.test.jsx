import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './footer'

describe('Partial: Footer', () => {
  test('Component is rendered', () => {
    const component = renderer.create(<Footer />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
