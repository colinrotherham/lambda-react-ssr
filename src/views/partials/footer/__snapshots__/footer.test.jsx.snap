// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Partial: Footer Component is rendered 1`] = `
<footer
  className="footer"
>
  <h2>
    Hello footer
  </h2>
  <p>
    Yes this is a footer
  </p>
</footer>
`;
